fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -220,6 +220,39 @@ then
 	log_Err_Abort "the --usejacobian option must be 'true' or 'false'"
 fi
 
+
+# ------------------------------------------------------------------------------
+#  Legacy Style Data Options
+# ------------------------------------------------------------------------------
+
+fMRIReference=`opts_GetOpt1 "--fmriref" $@`                          # reference BOLD run to use for movement correction target and to copy atlas registration from (or NONE; default)
+
+# Defaults
+fMRIReference=${fMRIReference:-NONE}
+
+
+# ------------------------------------------------------------------------------
+#  Compliance check
+# ------------------------------------------------------------------------------
+
+MPPMode=`opts_GetOpt1 "--mpp-mode" $@`
+MPPMode=`opts_DefaultOpt $MPPMode "HCPStyleData"`
+Compliance="HCPStyleData"
+ComplianceMsg=""
+
+# -- Use of BOLD reference
+
+if [ ! "${DoSliceTimeCorrection}" = 'NONE' ]; then
```

Comment:
This check appears to reference the wrong variable (one from a different PR entirely).  The message also references the t2w option.

Reply:
Indeed an error. Corrected now.